documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -101,7 +101,7 @@ For more information run
 
 ==== Writing to the eMMC (Windows)
 
-After `rpiboot` completes, a new USB mass storage drive will appear in Windows. We recommend following this xref:getting-started.adoc#installing-images-on-windows[guide] and using Win32DiskImager to write images to the drive, rather than trying to use `/dev/sda` etc. from Cygwin.
+After `rpiboot` completes, a new USB mass storage drive will appear in Windows. We recommend using https://www.raspberrypi.com/software/[Raspberry Pi Imager] to write images to the drive, rather than trying to use `/dev/sda` etc. from Cygwin.
```

Comment:
The "rather than trying to use `/dev/sda` etc. from Cygwin" part can probably be removed.

Reply:
Fixed.